sdk/js/src/api/http.js

```diff
@@ -141,7 +151,7 @@ class Http {
    * Extracts The Things Stack component abbreviation from the endpoint.
    *
    * @param {string} endpoint - The endpoint got for a request method.
-   * @returns {string} One of {is|as|gs|js|ns}.
+   * @returns {string} One of {is|as|gs|js|ns|edtc|qrg|gcs}.
```

Comment:
```suggestion
   * @returns {string} The stack component abbreviation.
```

We will also forget to update this next time for sure.